src/main/kotlin/com/autonomousapps/internal/Bundles.kt

```diff
@@ -92,6 +98,7 @@ internal class Bundles(private val dependencyUsages: Map<Coordinates, Set<Usage>
 
       // Handle bundles that don't have a primary entry point
       dependencyGraph.forEach { (_, view) ->
+        val projectNode = view.graph.nodes().find { it.identifier == projectPath }!!
```

Comment:
and the `!!` is appropriate because we know that the graph must have a node in it where the node's identifier matches the `projectPath`.

Reply:
Before, we were creating a Coordinates object and assumed that that exact object – according to `equals` – is in the graph. Now there is more information attached to the Coordinates – in particular the build name. So that did not work anymore.

I added a comment.